re-branding-the-default-login-page/authenticationendpoint/css/avis.css

```
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Change in other places too.

```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```